3rdparty/python/user_reqs.lock

```diff
@@ -1538,13 +1538,13 @@
         }
       ],
       "platform_tag": [
-        "cp39",
-        "cp39",
-        "macosx_11_0_arm64"
+        "cp310",
```

Comment:
I'm not sure its too important, but this bit of thrash is no longer necessary. I'll remove `platform_tag` on the next Pex release: https://github.com/pantsbuild/pex/issues/1685

Reply:
Great, thank you!